rpm/sailfish-secrets.spec

```diff
@@ -152,7 +152,6 @@ BuildRequires:  pkgconfig(glib-2.0)
 BuildRequires:  pkgconfig(gio-2.0)
 BuildRequires:  pkgconfig(gio-unix-2.0)
 Requires:  glib2
```

Comment:
Wouldn't this this `Requires:  glib2` be generated automatically during the build?